webapp/src/ts/modules/about/about.component.html

```diff
@@ -113,15 +117,19 @@
 
           <div>
             <label translate>replication.last.success.to</label>
-            <p *ngIf="!replicationStatus.disabled && replicationStatus.lastSuccessTo">{{replicationStatus.lastSuccessTo | simpleDateTime}}</p>
-            <p *ngIf="!replicationStatus.disabled && !replicationStatus.lastSuccessTo" translate>replication.last.success.unknown</p>
+            <p *ngIf="!replicationStatus.disabled && replicationStatus.lastSuccessTo">{{replicationStatus.lastSuccessTo
+              | simpleDateTime}}</p>
+            <p *ngIf="!replicationStatus.disabled && !replicationStatus.lastSuccessTo" translate>
+              replication.last.success.unknown</p>
             <p *ngIf="replicationStatus.disabled" translate>replication.disabled</p>
           </div>
 
           <div>
             <label translate>replication.last.success.from</label>
-            <p *ngIf="!replicationStatus.disabled && replicationStatus.lastSuccessFrom">{{replicationStatus.lastSuccessFrom | simpleDateTime}}</p>
-            <p *ngIf="!replicationStatus.disabled && !replicationStatus.lastSuccessFrom" translate>replication.last.success.unknown</p>
+            <p *ngIf="!replicationStatus.disabled && replicationStatus.lastSuccessFrom">
+              {{replicationStatus.lastSuccessFrom | simpleDateTime}}</p>
+            <p *ngIf="!replicationStatus.disabled && !replicationStatus.lastSuccessFrom" translate>
+              replication.last.success.unknown</p>
```

Comment:
There are a lot of line breaks being added which makes review difficult. Are these being added automatically by your editor?